### DISCUSSION TITLE
Rider GitHub Copilot Chat always show "You are not signed in to GitHub"

### DISCUSSION THREAD

**Discussion by JamesHuang1018** [community/community#76549] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

**IDE: JetBrains Rider 2023.2.3** 

I already signed in to GitHub and Copilot Status is Ready.

But when I open the GitHub Copilot Chat, it's always show "You are not signed in to GitHub".

**Copilot Status**
![image](https://github.com/community/community/assets/29995503/0a098f44-d23c-429c-9d98-1d60baf54d42)

**Copilot Chat Message**
![image](https://github.com/community/community/assets/29995503/e478b150-d5fb-4f5b-acc9-f8e66f6b3dc1)

**Comment by Akash1134** [comment, marked as answer]
Hi @JamesHuang1018 , Sorry for the trouble caused. This issue has been reported internally and we will update you with the next steps.

**Reply by JamesHuang1018** [reply]
Thanks

**Reply by ThatOssi** [reply]
any update or solution yet?

**Comment by PerfectlyFineCode** [comment]
I have the same issue, been like this for weeks.
Copilot Version: 1.4.17.4734

Rider Version: 2023.3.3 (Build #RD-233.14015.60, built on January 18, 2024)
Only Copilot Chat doesn't work, Copilot itself works fine and it accepts suggestions.

![image](https://github.com/community/community/assets/70201838/9219e2f9-b144-4e52-af2f-328b661ceff8)

**Comment by joseluisgs** [comment]
Me too when open Intellij o Rider. I need to click to login every time that I lunch de IDE
![Captura de pantalla 2024-09-17 184908](https://github.com/user-attachments/assets/fc92fe75-62f7-4cd4-ab1e-4740aec60736)

**Comment by davidweterings** [comment]
Same issue with Rider and Webstorm

**Comment by jeffrey-noel** [comment]
This issue is still happening almost two years later. It's impossible for me to use Copilot in any Jetbrains IDE. Is there any news about a possible fix?

**Comment by Badabum** [comment]
Still facing the same issue after getting licence for an organization

**Comment by abdalla-bushnaq** [comment]
I have created a new thread, as I noticed that I am using a different ide: intellij ultimate.
https://github.com/orgs/community/discussions/175527

I was using the github copilot free version successfully.
Now I updated to github copilot **enterprise subscription** and I get this problem.
**Copilot not usable in intellij since then**.
![Uploading ready.png…]()
![Uploading sign in to copilot.png…]()
